docs/security_conop/scans/SSG_STIG_SIMP_Client.rst

```diff
@@ -0,0 +1,1772 @@
+precludes the use of passwords to authenticate via ``sudo``.
+
+This may be configured differently and, by default, is restricted to
+only the ``administrators`` and ``security`` groups.
+
+.. code:: shell
+
+     cat /etc/sudoers | grep NOP
+    %administrators    ALL=(root) NOPASSWD:EXEC:SETENV: /bin/rm -rf /etc/puppetlabs/puppet/ssl
+    %administrators    ALL=(ALL) NOPASSWD:EXEC:SETENV: /usr/bin/sudosh
+    %administrators    ALL=(root) NOPASSWD:EXEC:SETENV: /usr/sbin/puppetca
+    %administrators    ALL=(root) NOPASSWD:EXEC:SETENV: /usr/sbin/puppetd
+    %security    ALL=(root) NOPASSWD:EXEC:SETENV: AUDIT
+
+--------------
+
+Disable Kernel Support for USB via Bootloader Configuration
+-----------------------------------------------------------
+
+-  Rule ID:
+   xccdf\_org.ssgproject.content\_rule\_bootloader\_nousb\_argument
+-  Type: Exception - Dangerous
+-  **Recommend SIMP Enhancement Request**
+
+Notes
+~~~~~
+
+Disabling global USB is *extremely* dangerous and will, most likely,
+cripple the ability to update systems and troubleshoot systems at all
+given that most modern systems no longer make USB keyboards and mice
+available.
+
+SIMP attempts to be sensible and disable block device connections
+instead.
+
+An enhancement request could be filed against SIMP to allow setting this
+kernel parameter but it should *not* be set by default unless no USB
+devices are detected on the system.
+
+.. code:: shell
+
+    cat /etc/modprobe.d/00_simp_blacklist.conf
+    # This file managed by Puppet.
+    install ieee1394 /bin/true
+    install usb-storage /bin/true
+
+--------------
+
+Ensure No Device Files are Unlabeled by SELinux
+-----------------------------------------------
+
+-  Rule ID:
+   xccdf\_org.ssgproject.content\_rule\_selinux\_all\_devicefiles\_labeled
+-  Type: False Positive
+-  **Requested feedback from the SSG team**
+
+Notes
+~~~~~
+
+This check simply appears to be broken
+
+--------------
+
+Direct root Logins Not Allowed
+------------------------------
+
+-  Rule ID:
+   xccdf\_org.ssgproject.content\_rule\_no\_direct\_root\_logins
+-  Type: Exception
+
+Notes
+~~~~~
+
+Removing all ability for Root to login from the console prevents "last
+effort" recovery of systems. This is not something that SIMP will enable
+by default.
+
+You can make this compliant by setting the following in Hiera:
+
+.. code:: yaml
+
+    ---
+    simplib::securetty : []
+
+--------------
+
+Restrict Serial Port Root Logins
+--------------------------------
+
+-  Rule ID:
+   xccdf\_org.ssgproject.content\_rule\_restrict\_serial\_port\_logins
+-  Type: Exception
+
+Justification
+~~~~~~~~~~~~~
+
+Removing all ability for Root to login from serial ports prevents "last
+effort" recovery of remote systems. This is not something that SIMP will
+enable by default.
+
+You can make this compliant by setting the following in Hiera:
+
+.. code:: yaml
+
+    ---
+    simplib::securetty :
```

Comment:
@trevor-vaughan is it better to just wait for the Beta before doing anything with this?